modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -268,7 +268,7 @@
 		//Media Item
 		renderMediaList: function(callback){
 			//Only render if medialist item are present
-			if (this.getTemplateData().length > 0) {
+			if (this.getTemplateData().length > (0 - !this.getConfig('showEmptyPlaylistError'))) {
 				//Generate new list template data
```

Comment:
@vladm3 - while this line does what is should, it is very unclear. Please try to avoid using boolean values inside arithmetic calculations, even if it means adding another line of code for checking the boolean condition. This will ensure readability in the feature. Thanks.